api/controllers/build.js

```diff
@@ -88,52 +88,55 @@ module.exports = {
   },
 
   status: (req, res) => {
-    let message;
+    let buildStatus;
 
-    const getStatusMessage = (statusRequest) => {
-      let statusMessage;
+    const getBuildStatus = (statusRequest) => {
+      let status;
+      let message;
       try {
-        statusMessage = decodeb64(statusRequest.body.message);
+        status = decodeb64(statusRequest.body.status);
```

Comment:
I don't think the status is being base64 encoded in the garden build

Reply:
goog catch .... i pushed this in garden build